webrtc.html

```diff
@@ -240,102 +242,58 @@ <h4><dfn>RTCIceCredentialType</dfn> Enum</h4>
               </tr>
               <tr>
                 <td><dfn data-idl><code>oauth</code></dfn></td>
-                <td><p>An OAuth 2.0 based authentication method, as described in
-                [[!RFC7635]]. It uses the OAuth 2.0 Implicit Grant type, with
-                PoP (Proof-of-Possession) Token type, as described in
-                [[!RFC6749]] and [[!OAUTH-POP-KEY-DISTRIBUTION]].
+                <td><p>An OAuth 2.0 based authentication method, as described
+                in [[!RFC7635]].
                 </p>
-                <p>The <dfn>OAuth Client</dfn> and the <dfn>Auhorization
-                Server</dfn> roles are defined in [[!RFC6749]] Section 1.1.
-                </p>
-                <p>
-                If [[!RFC7635]] is used in the WebRTC context then the <a>OAuth
-                Client</a> is responsible for refreshing the credential
-                information, and updating the <a>ICE Agent</a> with fresh new
-                credentials before the <code>accessToken</code> expires. The
-                <a>OAuth Client</a> can use the <a
-                href="#interface-definition"><code>RTCPeerConnection</code></a>
-                <a data-lt="setConfiguration"
-                href="#dom-rtcpeerconnection-setconfiguration"><code>
-                setConfiguration</code></a> method to periodically refresh the
-                TURN credentials. </p>
                 <p>For OAuth Authentication, the <a>ICE Agent</a> requires three
                 pieces of credential information. The credential is composed of
                 a <code>kid</code>, which the <a>RTCIceServer</a>
                 <code>username</code> member is used for, and
                 <code>macKey</code> and <code> accessToken</code>, which are
                 placed in the <a>RTCOAuthCredential</a> dictionary. All of this
                 information can be extracted from the OAuth response parameters,
-                which are received from the <dfn>Authorization Server</dfn>. The
+                which are received from the <a>Authorization Server</a>. The
                 relevant OAuth response parameters are the "kid", the "key", and
```

Comment:
These names ("kid","key","access_token") of the OAuth response parameters are from POP  key distribution draft, and may we should also remove to refer to them. According the actual changes we should rather state only that 

OAuth Client provide all necessary information (the <code>kid</code>, <code>macKey</code>, and <code>accessToken</code>) that are required by the <a>ICE Agent</a> for the Authentication.

But I think it is already mentioned some

Reply:
Good catch; removed everything in this paragraph from "All of this information..." onward. I think the "note" section below should clarify things sufficiently.